cdc/sink/manager.go

```diff
@@ -94,36 +86,8 @@ func (m *Manager) Close(ctx context.Context) error {
 	return nil
 }
 
-func (m *Manager) getMinEmittedTs(tableID model.TableID) model.Ts {
-	m.tableSinksMu.Lock()
-	defer m.tableSinksMu.Unlock()
-	if len(m.tableSinks) == 0 {
-		return m.getCheckpointTs(tableID)
-	}
-	minTs := model.Ts(math.MaxUint64)
-	for _, tblSink := range m.tableSinks {
-		resolvedTs := tblSink.getResolvedTs()
-		if minTs > resolvedTs {
-			minTs = resolvedTs
-		}
-	}
-	return minTs
-}
-
-func (m *Manager) flushBackendSink(ctx context.Context, tableID model.TableID) (model.Ts, error) {
-	// NOTICE: Because all table sinks will try to flush backend sink,
-	// which will cause a lot of lock contention and blocking in high concurrency cases.
-	// So here we use flushing as a lightweight lock to improve the lock competition problem.
-	if !atomic.CompareAndSwapInt64(&m.flushing, 0, 1) {
-		return m.getCheckpointTs(tableID), nil
-	}
-	m.flushMu.Lock()
```

Comment:
FYI,  mq sink is not thread-safe

Reply:
Manager flushes bufferSink concurrently, and bufferSink itself is thread-safe.